tests/regression/lp-1819728/task.yaml

```
@@ -0,0 +1,28 @@
+summary: Ensure that the systemctl from PR#8803 is fixed
+
+description: |
+    See https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1819728
+    for details about this issue.
+
+# TODO: enable on ubuntu-16.04-64, ubuntu-18.04-64 once the SRU
+#       for systemd is in the "-updates" pocket.
+systems: [ubuntu-core-16-64, ubuntu-core-18-64]
+
+kill-timeout: 2m
```

@sergiocazzolato suggested to use a 5 minute timeout to avoid issues on slower devices.

```suggestion
kill-timeout: 5m
```